workinggroups/README.md

```diff
@@ -0,0 +1,11 @@
+# CNCF Working Groups
+
+## Introduction
+
+The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing their findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a TOC member and voted with a super majority vote by the CNCF TOC. The TOC can also shut down a working group with a super majority vote.
+
+## Process
+
+If you would like to submit a working group proposal, please submit a pull request to the working groups folder. As an example, you can see the other working group proposals here: https://github.com/cncf/toc/tree/master/workinggroups
+
+You will also have to present to the CNCF TOC and wider community before your WG proposal will be voted upon by the TOC and community. You can request a presentation by filing an issue here: https://github.com/cncf/toc/issues
```

Comment:
## Expectations

Working Group representative(s) (either the TOC Sponsor or other representative) will provide updates to the TOC with some regularity (either as requested by the TOC or when meaningful WG milestone has been met).

Reply:
This is asking for both a pull-request (with the proposal) and an issue (for the presentation request).  Should those be rolled into a single pull-requests which carries that proposal and acts as a presentation request?  If not, what's the benefit to keeping them separate?

Reply:
For projects, the process has been to first request a presentation opportunity, and then, if invited to do so by the TOC, submit an official proposal. Most projects that have presented have not been asked to submit proposals.